analyzer/core/src/main/scala/io/scalac/newspaper/analyzer/core/Analyzer.scala

```diff
@@ -0,0 +1,53 @@
+package io.scalac.newspaper.analyzer.core
+
+import scala.collection.mutable
+
+class Analyzer {
+
+  import Analyzer._
+
+  private val archive: mutable.Map[String, PreprocessedContent] = new mutable.HashMap()
+
+  private val commentPattern = """(?s)<!--.*?-->""".r
+  private val tagPattern = """(?s)<(/?)\s*([a-zA-Z0-1-]+)[^>]*>""".r
+  private val scriptPattern = """(?s)<script>(.*?)</script>""".r
+  private val bodyPattern = """(?s)<body>(.*)</body>""".r
+
```

Comment:
What do you think about <iframe> are they still in use? Maybe we should ignore them as well?